registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/resources/bootstrap.properties

```diff
@@ -21,3 +23,6 @@ mosip.regproc.eventbus.kafka.commit.type=${mosip.regproc.uin.generator.eventbus.
 mosip.regproc.eventbus.kafka.max.poll.records=${mosip.regproc.uin.generator.eventbus.kafka.max.poll.records}
 mosip.regproc.eventbus.kafka.poll.frequency=${mosip.regproc.uin.generator.eventbus.kafka.poll.frequency}
 mosip.regproc.eventbus.kafka.group.id=${mosip.regproc.uin.generator.eventbus.kafka.group.id}
+
+
+uingenerator.lost.packet.update.information=phone,email
```

Comment:
dont add phone email here. Country might call phone as mobileNo in identity schema beacuse of which our code will fail. So dont assume any default value wherever identity schema is involved.

Reply:
ok done